src/icalendar/cal.py

```diff
@@ -17,9 +17,13 @@
 from icalendar.timezone_cache import _timezone_cache
 
 import pytz
+import six
 import dateutil.rrule
 from pytz.tzinfo import DstTzInfo
 
+if six.PY3:
+    unicode = str
```

Comment:
I don't think we are using six until now, but instead we have some of our own definitions in compat.py, please use that instead. Also, I'm somewhat confused why this works now, I don't see any other mention of ``unicode`` in the file?